leveler/commands/lvladmin/settings.py

```diff
@@ -148,6 +157,20 @@ async def lvlprivate(self, ctx):
             await self.config.guild(server).private_lvl_message.set(True)
             await ctx.send("**Private level-up alerts enabled for `{}`.**".format(server.name))
 
+    @checks.is_owner()
+    @lvladmin.command(name="antirepfarm")
```

Comment:
```suggestion
    @lvladmin.command()
```
Not needed, since command function is named same